web_explorer.py

```diff
@@ -56,24 +54,18 @@ def on_retriever_start(self, query: str, **kwargs):
     def on_retriever_end(self, documents, **kwargs):
         # self.container.write(documents)
         for idx, doc in enumerate(documents):
-            source = os.path.basename(doc.metadata["source"])
-            self.container.write(f"**Document {idx} from {source}**")
-            self.container.markdown(doc.page_content)
+            source = doc.metadata["source"]
+            self.container.write(f"**Results from {source}**")
+            self.container.text(doc.page_content)
 
 
 st.sidebar.image("img/ai.png")
 st.header("`Interweb Explorer`")
 st.info("`I am an AI that can answer questions by exploring, reading, and summarizing web pages."
     "I can be configured to use different moddes: public API or private (no data sharing).`")
 
-# Set keys (use for deployed Streamlit app)
-# For personal use, these can be hard-coded or set
-os.environ["GOOGLE_CSE_ID"] = st.secrets["GOOGLE_CSE_ID"]
-os.environ["GOOGLE_API_KEY"] = st.secrets["GOOGLE_API_KEY"]
-os.environ["OPENAI_API_KEY"] = st.secrets["OPENAI_API_KEY"]
-
```

Comment:
top level values in secrets.toml are automatically set in ENV so this is a no-op

Reply:
Ah, I see: I saw an earlier error related to missing env var in prod that appeared to require this.

But, glad it can be removed.